addons/mail/static/src/new/messaging_menu/messaging_menu.xml

```diff
@@ -32,7 +32,7 @@
                             onClick="(isMarkAsRead) => isMarkAsRead ? threadService.markAsRead(preview) : this.openDiscussion(preview)"
                         >
                             <t t-set-slot="icon">
-                                <PartnerImStatus t-if="preview.type === 'chat'" className="'position-absolute bottom-0 end-0'" partner="store.partners[preview.chatPartnerId]"/>
+                                <PartnerImStatus t-if="preview.type === 'chat'" className="'position-absolute bottom-0 end-0'" partner="store.personas[createLocalId('partner', preview.chatPartnerId)]"/>
```

Comment:
ok for later but can't we move this access up in the chain so that we work with record rather than id as much as possible?

Reply:
agree, follow-up PR